crates/ruff_cli/src/args.rs

```diff
@@ -76,12 +77,18 @@ pub enum Command {
 pub struct CheckCommand {
     /// List of files or directories to check.
     pub files: Vec<PathBuf>,
-    /// Attempt to automatically fix lint violations.
-    /// Use `--no-fix` to disable.
+    /// Apply fixes to resolve lint violations.
+    /// Use `--no-fix` to disable or `--unsafe-fixes` to include unsafe fixes.
```

Comment:
Should we say "potentially unsafe fixes"? (Defer to you.)

Reply:
Hm.. you were the one that told me unsafe is an all or nothing concept haha I'll ponder this as well.

Should it be "potentially incorrect fixes"? Fwiw I feel like `--unsafe-fixes` is the place for a more detailed description.

Reply:
Haha. I guess the way I thought about it was that a given fix is either safe or not. The collection of fixes may thereby contain unsafe changes.